submissions/EmmanuelUdoh/phpTask/create.php

```diff
@@ -0,0 +1,62 @@
+<?php
+if (isset($_POST['submit'])) {
+require "config.php";
+require "helper.php";
+
+    try {
+        $connection = new PDO($dsn, $username, $password, $options);
+
+        $newProduct = array(
+            "itemname" => $_POST['itemname'],
+            "quantity" => $_POST['quantity'],
+            "brand_id" => $_POST['brand'] + 1,
+        );
+
+        /**
+         * This is equivalent to
+         * INSERT INTO products (itemname, quantity) values (:itemname, :quantity)
+         */
+        $sql = sprintf(
+            "INSERT INTO %s (%s) values (%s)",
+            "products",
+            implode(", ", array_keys($newProduct)),
+            ":" . implode(", :", array_keys($newProduct))
+        );
+
+        $statement = $connection->prepare($sql);
+        $statement->execute($newProduct);
+    } catch (PDOException $error) {
+        echo $sql . "<br>" . $error->getMessage();
+    }
+}
+?>
+
+<?php if (isset($_POST['submit']) && $statement) { ?>
+    > <?php echo $_POST['itemname']; ?> successfully added.
+<?php }
+?>
+
+<h2>Add A Product </h2>
+
+<form method="post">
+<div class="form-container">
+    <label for="itemname">Name of Product</label>
+    <input type="text" name="itemname" id="itemname">
+    <?php session_start(); ?>
```

Comment:
In addition to the comments on the `fetch.php` file please remove a call to `session_start();`. Fetch all the brands in this file like it is being done in the `overview.php` file and use that instead.